api/_exclusivity/config.ts

```diff
@@ -0,0 +1,108 @@
+import * as sdk from "@across-protocol/sdk";
+import config from "../../src/data/exclusive-relayer-configs.json";
+import { RelayerConfig, RelayerSelector } from "./types";
+import { none, randomWeighted } from "./strategies";
+
+const { CHAIN_IDs } = sdk.constants;
+
+// Absolute minimum exclusivity to set for any given transfer.
+const DEFAULT_MIN_EXCLUSIVITY = 3;
+
+// For per-relayer configuration, filter by EXCLUSIVITY_MAX.
+// const EXCLUSIVITY_MAX = 5;
+
+/**
+ * Origin finality + destination confirmation delays are used initially for testing.
+ * These can likely be removed once actual per-route fill times are supplied by the data team.
+ */
+const DEFAULT_ORIGIN_FINALITY_DELAY = 1;
+const ORIGIN_FINALITY_DELAY: { [chainId: number]: number } = {
+  [CHAIN_IDs.MAINNET]: 18,
+  [CHAIN_IDs.LINEA]: 1.5,
+  [CHAIN_IDs.SCROLL]: 1.5,
+};
+
+const DEFAULT_DESTINATION_CONFIRMATION_DELAY = 1;
+const DESTINATION_CONFIRMATION_DELAY: { [chainId: number]: number } = {
+  [CHAIN_IDs.MAINNET]: 8,
+  [CHAIN_IDs.LINEA]: 1.5,
+  [CHAIN_IDs.SCROLL]: 1.5,
+};
+
+/**
+ * Object mapping named exclusivity strategies to their implementation.
+ */
+const EXCLUSIVITY_STRATEGIES = {
+  none,
+  randomWeighted,
+} as const;
+
+/**
+ * Select a specific relayer exclusivity strategy to apply.
+ * This currently hardcodes the "none" strategy, but will be updated to support additional strategies
+ * and selection from on env-based configuration.
+ * @returns A handler function that will narrow an array of relayers to a single address.
+ */
+export function getStrategy(): RelayerSelector {
```

Comment:
This function probably needs to accept route + token information to determine the correct strategy to apply based on config.

```suggestion
export function getStrategy(): RelayerSelector {
```